### PR TITLE
docs: Outreachy Blog1 - 2019 - Motivation to apply

### PR DESCRIPTION
**Pull request by nehagup**
Signed-off-by: nehagup

### CODE REVIEW

File: content/en/blog/news/outreachy-blog1-2019.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Outreachy - Motivation to apply! "
+date: 2019-01-10
```
Comment (MarckK) [inline content/en/blog/news/outreachy-blog1-2019.md]: 2020 -- Happy New Year! :)

Comment (nehagup) [reply content/en/blog/news/outreachy-blog1-2019.md]: oops!

**Pull request merged**